From: Antonin Kral <a.kral@bobek.cz>
Date: Thu, 24 Aug 2017 13:09:04 +0200
Subject: Set also Showmax specific auth header

---
 sso_cookie_auth.lua | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sso_cookie_auth.lua b/sso_cookie_auth.lua
index 13535a6..3fd8c2c 100644
--- a/sso_cookie_auth.lua
+++ b/sso_cookie_auth.lua
@@ -44,6 +44,7 @@ if cookie_auth_data ~= nil and cookie_auth_sign ~= nil then
       if tonumber(auth_data['exp']) >= ngx.time() then
         if auth_data['uid'] ~= cjson.null then
           ngx.req.set_header("X-Forwarded-User", auth_data['uid'])
+          ngx.req.set_header("showmax-int-Auth-Uid", auth_data['uid'])
         end
 
         -- Sanitize required audience
